recaptcha_enterprise/demosite/src/main/resources/static/styles/local.css

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Do we need an empty local.css file?

Reply:
Added by the UX engineer for testing, I believe. Will confirm and get back.

Reply:
This file is not needed anymore. Hence removing.